package.json

```diff
@@ -86,7 +86,7 @@
     "eslint-plugin-sorting": "^0.3.0",
     "eslint-plugin-standard": "^3.0.0",
     "eslint-plugin-unicorn": "^2.1.2",
-    "eslint-plugin-vue": "^2.0.1",
+    "eslint-plugin-vue": "^4.2.0",
```

Comment:
my only concern with such a big version bump is if there are breaking changes that would impact projects using the old version.  worth skimming the plugin changelog to see if anything jumps out.

if that seems ok, LGTM.